src/interface_global.pas

```diff
@@ -192,7 +193,6 @@ interface
      rep_Cuttings = Record
          Considered : BOOLEAN;
```

Comment:
Just happened to notice the uppercase letters, whereas all others are lowercase

Reply:
That's inconsistent in many places. Would keep it here and elsewhere for now since we dropped the interface anyway for now.